README.md

```diff
@@ -1,2 +1,32 @@
 # ui-plugin-find-eresource
-ERM-eresource-finder plugin for Stripes
+
+Copyright (C) 2017-2019 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License,
+Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
+
+## Introduction
+
+This package furnishes a single Stripes plugin of type `find-eresource`,
+which can be included in Stripes modules by means of a `<Pluggable
+type="find-eresource">` element. See [the *Plugins*
+section](https://github.com/folio-org/stripes-core/blob/master/doc/dev-guide.md#plugins)
+of the Module Developer's Guide.
+
+## Props
+
+| Name | Type | Description | Required |
+--- | --- | --- | --- |
+| `onEresourceSelected` | func: (eresource) => {} | Callback fired when a user clicks an eresource | Yes |
+| `dataKey` | string | Optional `dataKey` passed to stripes/connect when creating the connected Eresources component. |  |
+| `renderTrigger` | func: ({ triggerId, onClick, buttonRef }) => {} | Optional render function for the button to open the Eresource search modal. The `onClick` prop should be called when the trigger is clicked (assuming it is a Button). The `buttonRef` ensures that the trigger button is brought back into focus once the lookup modal is closed| |
+| `showPackages` | boolean | Optional prop to only display the packages eresources and its necessary filters | |
```

Comment:
This feels awkwardly-named imo. `showPackages` suggests to me that it controls whether packages are shown or not, in addition to other things. So the default (which isn't mentioned in this readme) would be `false` or `undefined`. So I'd assume that by default, this plugin shows only non-package resources. And if I want it to shown both, I pass `true` for this.

Reply:
Agreed...thinking if the prop could be a string such as `resourceType="Package"` instead? If not, extensibility wise we would have to add in additional boolean props in the future? for example: if we had to only display TitleInstances only or Journal or Books only, we would have to add one prop for each? what do you think?

Reply:
I'd say it's probably safer to just have an array like `showResourceTypes=['package']` if you want to go that route, rather than a string.

One thing that I think should be thought of here would be a scenario like if you were gonna start listing PCIs and how that affects users of this plugin. All of a sudden those users may have new resources available to them that the active app can't handle.

Reply:
And yeah, following on from your comment, what if someone wanted to show PCIs and Packages but not the Titles themselves. Props that are called `onlyShowFoo` would be confusing to them, but also may drive development of this plugin to not support such a use case.